Dockerfile

```diff
@@ -0,0 +1,16 @@
+### STAGE 1: Build ###
+FROM node:12.18.3 as build
```

Comment:
What's going on with this Docker stuff?

Reply:
It's the file that's used to create the Docker image. Should be the same one that we had in a working state with Kent, except he ran the build from his end last time.